types/google.visualization/index.d.ts

```diff
@@ -437,9 +437,15 @@ declare namespace google {
             zoomDelta?: number;
         }
 
-        export interface ChartStrokeFill {
+        export interface ChartStrokeFill extends ChartStroke, ChartFill {
```

Comment:
Might be a few other places to utilize this in the other charts.

Reply:
Yes, that's true, but it would require quite a bit of hunting in the docs, so I'd rather not do it in this PR